jetty-client/src/main/java/org/eclipse/jetty/client/AbstractHttpClientTransport.java

```diff
@@ -135,6 +135,7 @@ public void connect(InetSocketAddress address, Map<String, Object> context)
             catch (IOException xx)
             {
                 LOG.ignore(xx);
+                x.addSuppressed(xx);
```

Comment:
It's the usual exception that we must catch when we call `close()` - I think it can be ignored rather than added as suppressed.